### PR TITLE
Updated v1.14.0

### PR DESCRIPTION
**Pull request by mraspaud**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mraspaud** [comment]
@conda-forge-admin, please rerender

**Comment by mraspaud** [comment]
@djhoese @ocefpaf Any idea what went wrong ?

**Comment by djhoese** [comment]
@mraspaud I think this is related to some of the stuff you brought up on the pytroll slack with pyximports. It seems that the way you have the cython extensions set up it is trying to recompile or use the extension without having cython available in the build environment. Since it sounds like you've removed the cython dependency in master then doing a re-release with that fix should fix that here too. No idea why windows is the only failure here.

**Comment by mraspaud** [comment]
Well the C file is provided, so I didn't expect cython to be needed...

**Comment by djhoese** [comment]
So this error:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=105433&view=logs&j=171a126d-c574-5c8c-1269-ff3b989e923d&t=1e3b5e03-2b58-5acf-49d8-7964b3450f7e&l=824

```
    C:\Program Files (x86)\Common Files\Microsoft\Visual C++ for Python\9.0\VC\Bin\amd64\link.exe /DLL /nologo /INCREMENTAL:NO /LIBPATH:D:\bld\pyresample_1577097342813\_h_env\libs /LIBPATH:D:\bld\pyresample_1577097342813\_h_env\PCbuild\amd64 /LIBPATH:D:\bld\pyresample_1577097342813\_h_env\PC\VS9.0\amd64 /EXPORT:init_gradient_search build\temp.win-amd64-2.7\Release\pyresample/gradient/_gradient_search.obj /OUT:build\lib.win-amd64-2.7\pyresample\gradient\_gradient_search.pyd /IMPLIB:build\temp.win-amd64-2.7\Release\pyresample/gradient\_gradient_search.lib /MANIFESTFILE:build\temp.win-amd64-2.7\Release\pyresample/gradient\_gradient_search.pyd.manifest
    _gradient_search.obj : warning LNK4197: export 'init_gradient_search' specified multiple times; using first specification
       Creating library build\temp.win-amd64-2.7\Release\pyresample/gradient\_gradient_search.lib and object build\temp.win-amd64-2.7\Release\pyresample/gradient\_gradient_search.exp
    _gradient_search.obj : error LNK2019: unresolved external symbol isinf referenced in function __pyx_f_10pyresample_8gradient_16_gradient_search_one_step_gradient_search_no_gil
    build\lib.win-amd64-2.7\pyresample\gradient\_gradient_search.pyd : fatal error LNK1120: 1 unresolved externals
    error: command 'C:\\Program Files (x86)\\Common Files\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\amd64\\link.exe' failed with exit status 1120
    Running setup.py install for pyresample: finished with status 'error'
```

Looks like it can't find `isinf` and is talking about the `.pyd` file.

**Comment by ocefpaf** [comment]
There is no isinf in vs9

We can patch it or drop vc9 support (py27 on Windows).

**Comment by mraspaud** [comment]
dropping python 2.7 on windows is tempting. How much work would the patch need ?

**Comment by ocefpaf** [comment]
> dropping python 2.7 on windows is tempting.

:+1: to that.

> How much work would the patch need ?

Probably [something like this](https://github.com/conda-forge/gmpy2-feedstock/pull/12/commits/aeaa8ff08d95cef06e3a5fec891b4e12d864d8ed#diff-f89afe594ea1e92963a26d8a464759df).

**Comment by mraspaud** [comment]
@djhoese what do you say about disabling python 2.7 in windows ? we're going to remove python 2 support in the next release anyway...

File: .azure-pipelines/azure-pipelines-win.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ jobs:
...
     maxParallel: 4
     matrix:
-      win_c_compilervs2008cxx_compilervs2008python2.7:
-        CONFIG: win_c_compilervs2008cxx_compilervs2008python2.7
-        CONDA_BLD_PATH: D:\\bld\\
-        UPLOAD_PACKAGES: True
+      # win_c_compilervs2008cxx_compilervs2008python2.7:
+      #   CONFIG: win_c_compilervs2008cxx_compilervs2008python2.7
+      #   CONDA_BLD_PATH: D:\\bld\\
+      #   UPLOAD_PACKAGES: True
```
Comment (ocefpaf) [inline .azure-pipelines/azure-pipelines-win.yml]: We don't edit the CI config directly. Let me add the skip statement here for you.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by mraspaud** [comment]
@ocefpaf thanks for fixing my mistake !
@djhoese I'm merging this, as we don't even run the pyresample unit tests on python 2.7

**Comment by djhoese** [comment]
Sorry, I was purposely trying to avoid the internet over the holidays. This is all good with me. Thanks for all the work you two.

**Pull request merged**